core/debugger/remote_debugger.cpp

```diff
@@ -237,7 +237,8 @@ void RemoteDebugger::_print_handler(void *p_this, const String &p_string, bool p
 
 		OutputString output_string;
 		output_string.message = s;
-		output_string.type = p_error ? MESSAGE_TYPE_ERROR : MESSAGE_TYPE_LOG;
+		output_string.type = p_error ? MESSAGE_TYPE_ERROR : p_rich ? MESSAGE_TYPE_LOG_RICH
+																   : MESSAGE_TYPE_LOG;
```

Comment:
Avoid nesting ternary operators, as it's not very readable:

```suggestion
		if (p_error) {
			output_string.type = MESSAGE_TYPE_ERROR;
		} else if (p_rich) {
			output_string.type = MESSAGE_TYPE_LOG_RICH;
		} else {
			output_string.type = MESSAGE_TYPE_LOG;
		}
```

Reply:
Sorry, I thought less lines is better. ^^"